http/jaxrs-reactive/src/main/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+quarkus.openshift.route.expose=true
```

Comment:
Honestly, I don't really know why this property is needed, but the scenario fails without it.

Reply:
I think it makes sense as otherwise the route is not exposed and hence you can't verify your endpoints.

Reply:
That's the weird thing, the endpoints work fine without it, because the test framework itself contains this setting. But some of the tests fail because of https://access.redhat.com/solutions/4820731.